derrida/books/management/commands/reference_data.py

```diff
@@ -36,6 +36,9 @@ def add_arguments(self, parser):
             '-d', '--directory',
             help='Specify the directory where files should be generated')
 
+    def remove_empty_keys(self, list_of_dicts):
+        return [{field: d[field] for field in d.keys() if d[field] not in [None, '', []]} for d in list_of_dicts]
```

Comment:
```suggestion
    def remove_empty_keys(self, list_of_dicts):
        return [OrderedDict([(key, val) for key, val in d.items() if val])
                for d in list_of_dicts]
```

Reply:
I was testing locally and noticed the JSON output order looked nothing like what you have in the instance data method.  Figured out the culprit was your filter method, which converts it from an OrderedDict to a regular dict, and this project is (😞) on a version of python where that still matters.

Here's a solution that resolves the output order.

Reply:
@rlskoeser Okay! I'll cast it as a list of OrderedDicts. Your suggestion would fail the tests we set up. Were you just writing shorthand or did you intentionally remove `val not in [None, '', []]`

Reply:
I checked locally and it passed the tests for me. `if val` is equivalent to your more verbose check, because `None`, empty string, and empty list are all false

Reply:
✅ 08ae332ed0e7218799971b0445900bc2865682bc